src/main/java/nextstep/optional/User.java

```diff
@@ -33,7 +36,11 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        return Optional.ofNullable(user)
+                .map(User::getAge)
+                .filter(age -> age >= 30)
```

Comment:
```suggestion
                .filter(age -> age >= 30 && age <= 45)
```
로 바꾸면 filter 를 한번 덜 거칠 수 있을 것 같습니다!